Gems/Atom/RHI/Vulkan/Code/Source/RHI/MemoryPageAllocator.cpp

```diff
@@ -49,9 +49,11 @@ namespace AZ
 
         void MemoryPageFactory::ShutdownObject(Memory& memory, bool isPoolShutdown)
         {
+/*          // Allocation statistics are broken with VMA change
             RHI::HeapMemoryUsage& heapMemoryUsage = *m_descriptor.m_getHeapMemoryUsageFunction();
             heapMemoryUsage.m_totalResidentInBytes -= memory.GetDescriptor().m_sizeInBytes;
             heapMemoryUsage.Validate();
+*/
```

Comment:
This code should be removed or a GHI should be made to address fixing htis.

Reply:
I haven't removed this yet. Basic statistics work in straightforward cases, but seem to be broken in others (e.g. going into game mode in the editor). I'm guessing it was removed because the tracking is being done elsewhere.